frontend/web/styles/3rdParty/_hljs.scss

```
@@ -256,18 +241,18 @@ code.txt {
   font-size: $font-size-base;
   font-weight: 500;
   background: transparent !important;
-  border: 1px solid $input-border-color;
+  border: 1px solid var(--color-border-default);
   font-family: 'OpenSans', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "OpenSans" (font-family-name-quotes)

(font-family-name-quotes)